frontend/src/app/api/searchFetcher.ts

```diff
@@ -35,43 +34,32 @@ const filterNameMap = {
   category: "funding_category",
 } as const;
 
-export default class SearchOpportunityAPI extends BaseApi {
-  get namespace(): string {
-    return "opportunities";
-  }
-
-  async searchOpportunities(searchInputs: QueryParamData) {
-    const { query } = searchInputs;
-    const filters = buildFilters(searchInputs);
-    const pagination = buildPagination(searchInputs);
+export const searchForOpportunities = async (searchInputs: QueryParamData) => {
```

Comment:
kept the necessary custom search related functionality from the SearchOpportunityApi here